index.ts

```diff
@@ -99,6 +100,8 @@ try {
 
     if (isYarn) {
       await exec('yarn', ['install', '--frozen-lockfile']);
+    } else if (isPnpm) {
+      await exec('pnpm', ['install']);
```

Comment:
```suggestion
      await exec('pnpm', ['install', '--frozen-lockfile']);
```

https://pnpm.io/cli/install#--frozen-lockfile